test/integration/servergroups_integration_test.go

```diff
@@ -36,13 +37,56 @@ func TestIntegrationServerGroup_CRUD(t *testing.T) {
 
 func createServerGroup(t *testing.T) (*cloudscale.ServerGroup, error) {
 	createRequest := &cloudscale.ServerGroupRequest{
-		Name:         serverBaseName + "-group",
-		Type:         "anti-affinity",
+		Name: serverBaseName + "-group",
+		Type: "anti-affinity",
 	}
 
 	return client.ServerGroups.Create(context.Background(), createRequest)
 }
 
+func TestIntegrationServerGroup_MultiSite(t *testing.T) {
+	integrationTest(t)
+
+	allZones := getAllZones(t)
+	if len(allZones) <= 1 {
+		t.Skip("Skipping MultiSite test.")
+	}
+
+	var wg sync.WaitGroup
+
+	for _, zone := range allZones {
+		wg.Add(1)
+		go createServerGroupInZoneAndAssert(t, zone, &wg)
+	}
+
+	wg.Wait()
+}
+
+func createServerGroupInZoneAndAssert(t *testing.T, zone cloudscale.Zone, wg *sync.WaitGroup) {
+	defer wg.Done()
+
+	createServerGroupRequest := &cloudscale.ServerGroupRequest{
+		Name: "Yellow Submarine",
+		Type: "anti-affinity",
+	}
+
+	createServerGroupRequest.Zone = zone.Slug
```

Comment:
AFAIK this attribute does not exist. Also you could just assign it above instead of here.

Reply:
Added attribute (see above). I have assigned it here since the short-hand syntax for embedded types can be used. It's a matter of taste I'd say.